docs/tutorials/using-host-labels.md

```diff
@@ -0,0 +1,204 @@
+# Use host labels to organize systems, metrics, and alarms
+
+When you use Netdata to monitor and troubleshoot an entire infrastructure, whether that's dozens or hundreds of systems,
+you need sophisticated ways of keeping everything organized. You need alarms that adapt to the system's purpose, or
+whether the `master` or `slave` in a streaming setup. You need properly-labeled metrics archiving so you can sort,
+correlate, and mash-up your data to your heart's content. You need to keep tabs on ephemeral Docker containers in a
+Kubernetes cluster.
+
+You need **host labels**: a powerful new way of organizing your Netdata-monitored systems. We introduced host labels in
+[v1.20 of Netdata](https://blog.netdata.cloud/posts/release-1.20/), and they come pre-configured out of the box.
+
+Let's take a peek into how to create host labels and apply them across a few of Netdata's features to give you more
+organization power over your infrastructure.
+
+## Create unique host labels
+
+Host labels are defined in `netdata.conf`. To create host labels, open that file using `edit-config`.
+
+```bash
+cd /etc/netdata   # Replace this path with your Netdata config directory, if different
+sudo ./edit-config netdata.conf
+```
+
+Create a new `[host labels]` section wherever you choose:
+
+```conf
+[host labels]
+```
+
+Beneath that section, you can define a new host label, and its value for the system in question, taking care not to
+violate any of the [host label naming rules](../configuration-guide.md#netdata-labels), like so:
+
+```conf
+[host labels]
+    type = webserver
+    location = us-seattle
+    provider = aws
+```
+
+Once you've written a few host labels, you need to enable them. Instead of restarting the entire Netdata service, you
+can reload labels using the helpful `netdatacli` tool:
+
+```bash
+netdatacli reload-labels
+```
+
+Your host labels will now be enabled. You can double-check these by using `curl http://HOST-IP:19999/api/v1/info` to
+read the status of your agent. For example, from a VPS system running Debian 10:
+
+```json
+{
+  ...
+	"host_labels": {
+	  "_is_master": "false",
+		"_virt_detection": "systemd-detect-virt",
+		"_container_detection": "none",
+		"_container": "unknown",
+		"_virtualization": "kvm",
+		"_architecture": "x86_64",
+		"_kernel_version": "4.19.0-6-amd64",
+		"_os_version": "10 (buster)",
+		"_os_name": "Debian GNU/Linux",
+		"type": "webserver",
+    "location": "seattle",
+    "provider": "aws"
+	},
+  ...
+}
+```
+
+You may have noticed a handful of labels that begin with an underscore (`_`). These are automatic labels.
+
+### Automatic labels
+
+When Netdata starts, it captures relevant information about the system and converts them into automatically-generated
+host labels. You can these use these to logically organize your systems via health entities, exporting metrics,
```

Comment:
I think, there is an excess of `these` in the sentence `You can these use these to logically`